### ISSUE TITLE
Add info about automated tools to accessibility criteria document

### ISSUE THREAD

**Issue by EoinShaughnessy** [alphagov/govuk-frontend#2357]
## What

Based on user feedback, add info about automated tools to [accessibility criteria doc](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md).

## Why

After we got 2i approval to publish our accessibility criteria, we realised it lacked content about automated tools that help detect accessibility issues.

### Points to consider

- according to [GOV.UK's blog on testing with a deliberately inaccessible webpage](https://accessibility.blog.gov.uk/2017/02/24/what-we-found-when-we-tested-tools-on-the-worlds-least-accessible-webpage/), 29% of the accessibility barriers went undetected by any of the 10 automated tools used - for more info, see the section titled 'Lots of the barriers weren’t found by any of the tools'
- however, automated tools (for example, Axe, SiteImprove) are still useful for picking up basic issues
- we'd like to hear from the Design System community about the automated tools they find useful

## Who needs to know about this

Technical Writer, Community Manager, Developers

## Done when

- [ ] Developer and Technical Writer draft update
- [ ] Update receives review from teammate
- [ ] Update passes 2i
- [ ] We publish update

**Comment by EoinShaughnessy** [comment]
@abbott567 That helps massively - thanks!

@hannalaakso FYI 👀
